### DISCUSSION TITLE
First experience with this theme :-)

### DISCUSSION THREAD

**Discussion by ilg-ul** [citkane/typedoc-theme-yaf#7] [General]
After fixing the absolute symlinks reported in #6, I successfully published a site using this theme:

- ~~<https://xpack.github.io/logger-ts/stable/>~~

[2023-01-21 update] This project did not pass the 24 hour response time test; the project site was regenerated with the default TypeDoc theme. :-(

**Comment by citkane** [comment]
Hi @ilg-ul

This project is not my full time job, I do not get any payment for it.
I would love to be able to respond within 24h like I was paid to be a professional help-desk, but I simply can't.

I am busy with many things.
This project is important to me, and will get the attention it needs appropriately.
